packages/shared/src/react/contexts.tsx

```diff
@@ -50,6 +50,23 @@ const OrganizationProvider = ({
   );
 };
 
+function useAssertWrappedByClerkProvider(displayNameOrCustomHandler?: string | (() => void)): void {
+  const ctx = React.useContext(ClerkInstanceContext);
+
+  if (!ctx) {
+    if (typeof displayNameOrCustomHandler === 'function') {
```

Comment:
🔧 Since there is currently no usage of `displayNameOrCustomHandler` as a function i would omit this. Based on this PR there is no case of passing a function as a parameter.

Reply:
@dimkl It's used here: https://github.com/clerk/javascript/pull/2299/files#diff-f023857f9a8061e1b588df1ac38d27994ca07359dbf77a99b53bc44fc3f82fda